trie/trie.go

```diff
@@ -60,6 +62,10 @@ func (t *Trie) Db() *Database {
 	return t.db
 }
 
+func (t *Trie) SetDb(db *Database) {
```

Comment:
exported func should be have comments for godoc

Reply:
all comments have been added :)